nuc_morph_analysis/lib/visualization/write_data_for_colorizer.py

```diff
@@ -419,7 +417,7 @@ def make_features(
         if scale_factor is not None:
             data = data * scale_factor
             
-        description = GLOSSARY[feature.column_name]
+        description = GLOSSARY.get(feature.column_name, "Coming soon")
```

Comment:
If you wanted, it's okay also to leave this blank (`""`). I think `Coming soon` is fine too, just another option.

![image](https://github.com/user-attachments/assets/76abbcd2-b5dc-404c-8371-1b1678016d54)